tests/expected/matlab/functions_wrapper.cpp

```diff
@@ -130,20 +130,59 @@ void DefaultFuncInt_8(int nargout, mxArray *out[], int nargin, const mxArray *in
   int b = unwrap< int >(in[1]);
   DefaultFuncInt(a,b);
 }
-void DefaultFuncString_9(int nargout, mxArray *out[], int nargin, const mxArray *in[])
+void DefaultFuncInt_9(int nargout, mxArray *out[], int nargin, const mxArray *in[])
+{
+  checkArguments("DefaultFuncInt",nargout,nargin,1);
+  int a = unwrap< int >(in[0]);
+  DefaultFuncInt(a,0);
+}
+void DefaultFuncInt_10(int nargout, mxArray *out[], int nargin, const mxArray *in[])
+{
+  checkArguments("DefaultFuncInt",nargout,nargin,1);
+  int b = unwrap< int >(in[0]);
+  DefaultFuncInt(123,b);
```

Comment:
Based on my previous comment, I don't think this scenario will ever happen, unless you have python-like keyword arguments.

Reply:
Ya - see above comment that I think the added flexibility is worth the tradeoff that we will get some blocks that will never get called.